modules/pop3/scanner.go

```diff
@@ -179,6 +178,8 @@ func VerifyPOP3Contents(banner string) zgrab2.ScanStatus {
 	     strings.Contains(lowerBanner, "spamhaus"),
 	     strings.Contains(lowerBanner, "relay"):
 		return zgrab2.SCAN_SUCCESS
+	case banner == "+OK\r\n":
```

Comment:
minor nit: since it seems we're just concatenating all cases that would lead to `SCAN_SUCCESS`, let's just adds this as a case to the above `case` statement

Reply:
7eee4ffe0a1be7ab12c6296cf0b40081cc0cb273